### PR TITLE
ENH Update folders + namespaces, remove unused class

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-session-manager/issues/42

### CODE REVIEW

**Review by bergice — CHANGES_REQUESTED** [review]
Causes a fatal error if there are `LoginSessions` existing already before applying this as it's processing them in the middleware: `[Emergency] Uncaught SilverStripe\ORM\ValidationException: Object is of class 'SilverStripe\SessionManager\Model\LoginSession' which doesn't exist - you need to change the ClassName before you can write it`.
Might be okay if no one is using the module yet.

After removing `LoginSessions` from the database, I'm getting this error: `[Emergency] Uncaught Error: Call to undefined method SilverStripe\Security\RememberLoginHash::setLogoutAcrossDevices()`. Not sure why this method is gone.
Seems like it should now be `RememberLoginHash::config()->update('logout_across_devices', false);`

**Comment by emteknetnz** [comment]
@bergice you probably need to run `composer update` - basically you want to be runing 4.8 on your local - there was new API added to framework `RememberLoginHash::setLogoutAcrossDevices()` and the requirement withing session-manager was bumped to `"silverstripe/framework": "^4.8",`

The [new API](https://github.com/silverstripe/silverstripe-framework/commit/bcccc63d33485b6ff434f8ce8e4cbc4e3b099e8c#diff-8282d5ad2e6bb6e4f5bd7b05307a79dc50130d0dd9bcf8a8d9ecc696f74c462b) was added specifically so we did not run `RememberLoginHash::config()->update('logout_across_devices', false)` which is messy as you're changing a framework config directly from a satellite module

File: src/Controllers/LoginSessionController.php [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
+use SilverStripe\Control\Controller;
 use SilverStripe\Control\HTTPRequest;
 use SilverStripe\Control\HTTPResponse;
 use SilverStripe\Security\SecurityToken;
-use SilverStripe\SessionManager\Model\LoginSession;
+use SilverStripe\SessionManager\Models\LoginSession;
 
-// TODO: move this to Controllers folder and extend Controller instead of LeftAndMain
-class LoginSessionController extends LeftAndMain
+class LoginSessionController extends Controller
```
Comment (maxime-rainville) [inline src/Controllers/LoginSessionController.php:12]: @emteknetnz @bergice Changing this class from `LeftAndMain` to `Controller` completely nukes the functionality. I understand this is a big PR, but you should at least try to run these things before merging them.

Comment (maxime-rainville) [reply src/Controllers/LoginSessionController.php:12]: I'll fix it through https://github.com/silverstripe/silverstripe-session-manager/issues/67

Comment (maxime-rainville) [reply src/Controllers/LoginSessionController.php:12]: Sorry, it was my screw up. I change something else on my set up ... just ignore me.

**Pull request merged**